infra.bs

```diff
@@ -297,8 +297,8 @@ a [=byte sequence=] <var ignore>bytes</var>, perform the following steps. They r
 [=string=] or null.
 
 <p>Algorithms which do not return a value use a shorter form. This same shorter form can be used even
-for algorithms that do return a value, since the return type is relatively easy to infer from the
-algorithm steps and call sites:
+for algorithms that do return a value if the return type is relatively easy to infer from the
+algorithm steps:
```

Comment:
Comma still seems good? Not entirely sure, but it's a pretty long sentence otherwise.

I was also thinking that we could say "nearby callers", but if you have to look at callers at all it's probably best clarified anyway.

Reply:
I removed the comma because before it was ok to skip the second half of the sentence, but now it's not ok. But I don't care very much. :)